.github/workflows/phpunit-tests.yml

```diff
@@ -27,14 +26,6 @@ jobs:
   #
   # Performs the following steps:
   # - Cancels all previous workflow runs for pull requests that have not completed.
-  # - Checks out the repository.
-  # - Logs debug information about the runner container.
-  # - Installs NodeJS 14.
-  # - Sets up caching for NPM.
-  # _ Installs NPM dependencies using install-changed to hash the `package.json` file.
-  # - Builds WordPress to run from the `build` directory.
-  # - Creates a ZIP file of compiled WordPress.
-  # - Uploads ZIP file as an artifact.
   setup-wordpress:
```

Comment:
Let's change the slug and name of the job to something more accurate. Maybe `setup-tasks` and `Setup workflow`?